internal/provider/schema_dlp_profiles.go

```diff
@@ -0,0 +1,99 @@
+package provider
```

Comment:
let's rename this file for consistency too; `schema_dlp_profiles.go` -> `schema_cloudflare_dlp_profiles.go`